contracts/src/lido/README.md

```diff
@@ -0,0 +1,105 @@
+
+- Bridging of Lido's tokens from L1 to L2 chains
+- Instant ETH staking on L2 chains with receiving stETH/wstETH on the corresponding L2 immediately
+- Keeping UX on L2 as close as possible to the UX on Ethereum mainnet
+
+At this point, the implementation must provide a scalable and reliable solution for Lido to bridge ERC20 compatible tokens between Scroll and the Ethereum chain.
+
+[^*]: The current implementation might not support the non-standard functionality of the ERC20 tokens. For example, rebasable tokens or tokens with transfers fee will work incorrectly. In case your token implements some non-typical ERC20 logic, make sure it is compatible with the bridge before usage.
+
+## Security surface overview
+
+| Statement                                                                                                                                    | Answer |
+| -------------------------------------------------------------------------------------------------------------------------------------------- | ------ |
+| It is possible to bridge wstETH forth and back using this bridge                                                                             | Yes    |
+| The bridge using a canonical mechanism for message/value passing                                                                             | Yes    |
+| The bridge is upgradeable                                                                                                                    | Yes    |
+| Upgrade authority for the bridge                                                                                                             | TBA    |
+| Emergency pause/cancel mechanisms and their authorities                                                                                      | TBA    |
+| The bridged token support permits and ERC-1271                                                                                               | Yes    |
+| Are the following things in the scope of this bridge deployment:                                                                             |        |
+| - Passing the (w)stETH/USD price feed                                                                                                        | No     |
+| - Passing Lido DAO governance decisions                                                                                                      | TBA    |
+| Bridges are complicated in that the transaction can succeed on one side and fail on the other. What's the handling mechanism for this issue? | TBA    |
+| Is there a deployment script that sets all the parameters and authorities correctly?                                                         | TBA    |
```

Comment:
Will you add the deployment script in a follow-up PR? Should we just include the deployment script in this PR directly?

Reply:
would be helpful to have

Reply:
the deploy script is added. But the upgrade process will involve multisig operation by Scroll. I suggest to add that later.

Reply:
sure thing